requirements-container.txt

```diff
@@ -1,5 +1,5 @@
 napalm==3.3.1
 ruamel.yaml==0.17.16
 django-auth-ldap==3.0.0
-google-crc32c==1.1.2
+google-crc32c==1.1.4
```

Comment:
I think we don't need the version fixed, when we are building the C library.

Reply:
We don't, true. But since we build the c lib just because of this dependency, I'd rather have it. Then we see how it evolves, because Renovate will push updates to it. And if it will ever be changed in a way so that it can compile the C code itself on Alpine as well, then we can also remove the build step again in the Dockerfile.